instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+import instaheart from './Images/instaheart.png';
+import comment from './Images/comment.png';
+
+import './CommentSection.css';
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      newComment: '',
+      likes: 0
+    };
+  }
+
+  addNewComment = event => {
+    event.preventDefault();
+    this.setState({
+      comments: [
+        ...this.state.comments,
+        { text: this.state.newComment, username: 'MackDaddy' }
+      ],
+      newComment: ''
+    });
+  };
+
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  incrementLikes = event => {
+    event.preventDefault();
+    this.setState({ likes: this.state.likes + 1 });
```

Comment:
it seems odd that the likes just increment without end. typically you would be allowed to like/unlike on each click.